dailystats.py

```diff
@@ -50,87 +51,91 @@ def read_stats_file(csv_file_name):
+                casecount = 0
+            else:
+                casecount = int(dategroup[0][1])
+
+            if date_object > PEAK_DATE_OBJECT:
+                # WE ARE INTERESTED (results should be improving )
+                recent_total = recent_total + casecount
+                recent_count += 1
+            elif date_object == PEAK_DATE_OBJECT:
+                # We are interested - this should be the peak case count
+                peak_case_count = casecount
+            else:
+                # We are not interested - these dates are before the peak / lockdown start
+                pass
+
+        if recent_count > 0:
+            recent_average = recent_total / recent_count
+        if recent_average < 1:
+            recent_average = 1
+
+        # TODO replace this logic - compute the weekly average for the peak week instead of using a single day
+        # TODO instead of overall average since peak date, use rolling 7 day average
+        progress_factor = peak_case_count / recent_average
+
+        print("{:<3}: Region ==> {:<30} peak count ==> {:<5} recent average is ==> {:7.1f}  progress_factor is ==> {:3.2f}"
+              .format(region_count, region, peak_case_count, recent_average, progress_factor))
+        output_line = "{:<30},{:<5},{:7.1f},{:3.2f}\n".format(region, peak_case_count, recent_average, progress_factor)
+        results_file.write(output_line)
+
+        if recent_average > peak_case_count:
             pass
+            # print(" {}: HIGH ALARM For Region ==> {} peak count is ==> {} recent average is ==>{:10.2f}"
+            # .format(region_count, region, peak_case_count, recent_average))
+    results_file.close()
+    print("\nOutput Results have been written to {} ".format(OUTPUT_FILE_NAME))
 
-    if recent_count > 0:
-        recent_average = recent_total / recent_count
-    if recent_average < 1:
-        recent_average = 1
 
-    # TODO replace this logic - compute the weekly average for the peak week instead of using a single day
-    # TODO instead of overall average since peak date, use rolling 7 day average
-    progress_factor = peak_case_count / recent_average
-
-    print("{:<3}: Region ==> {:<30} peak count ==> {:<5} recent average is ==> {:7.1f}  progress_factor is ==> {:3.2f}"
-          .format(region_count, region, peak_case_count, recent_average, progress_factor))
-    output_line = "{:<30},{:<5},{:7.1f},{:3.2f}\n".format(region, peak_case_count, recent_average, progress_factor)
-    results_file.write(output_line)
-
-    if recent_average > peak_case_count:
-        pass
-        # print(" {}: HIGH ALARM For Region ==> {} peak count is ==> {} recent average is ==>{:10.2f}"
-        # .format(region_count, region, peak_case_count, recent_average))
-results_file.close()
-print("\nOutput Results have been written to {} ".format(OUTPUT_FILE_NAME))
-
-# TODO: For England and poorly performing regions produce graphical output
+if __name__ == "__main__":

```

Comment:
`__name__` is a special variable which shows where this is being executed. If you run a module as a script, then `__name__ == "__main__"`, otherwise it would be `"dailystats"`